install/linux/usr/share/odemis/hwtest/orsay.odm.yaml

```diff
@@ -52,3 +56,18 @@ Orsay: {
     role: gis-reservoir,
     init: {}
 }
+
+"FIB-vacuum": {
+    role: fib-vacuum,
+    init: {}
+}
+
+"FIB-source": {
+    role: fib-source,
```

Comment:
@pieleric These roles are currently not defined in odemis/gui/model/__init__.py.
Do we want to make the roles already similar as for the FIB of Enzel?
And should this PR already these roles to __init__.py? (I don't think the latter necessarily)

Reply:
Most probably this role will go away, once we have extra wrappers. For now it's fine. It's known that the GUI cannot use this backend. We'll care about the GUI way later ;-)